src/raymath.h

```diff
@@ -246,7 +246,7 @@ RMDEF float Vector2Distance(Vector2 v1, Vector2 v2)
 // Calculate angle from two vectors in X-axis
 RMDEF float Vector2Angle(Vector2 v1, Vector2 v2)
 {
-    float result = atan2f(v2.y - v1.y, v2.x - v1.x)*(180.0f/PI);
+    float result = (float)atan2f(v2.y - v1.y, v2.x - v1.x)*(180.0f/PI);
```

Comment:
atan2f() should already return a float...

Reply:
Yes this, and the others were actually a problem with PI that I miss diagnosed and didn't revert after I fixed PI..